cli/i18n/I18nData.php

```diff
@@ -318,36 +306,35 @@ public function ignore($key, $language, $reverse = false) {
 
 	/**
 	 * Ignore all unmodified keys from a language, or reverse it.
-	 *
-	 * @param string $language
-	 * @param boolean $reverse
 	 */
-	public function ignore_unmodified($language, $reverse = false) {
+	public function ignore_unmodified(string $language, bool $reverse = false): void {
 		$my_language = $this->getLanguage($language);
 		foreach ($this->getReferenceLanguage() as $file => $ref_language) {
 			foreach ($ref_language as $key => $ref_value) {
 				if (array_key_exists($key, $my_language[$file])) {
-					if($ref_value->equal($my_language[$file][$key])) {
+					if ($ref_value->equal($my_language[$file][$key])) {
```

Comment:
`$ref_value` does not look typed here. Again a sign that the signature of `getReferenceLanguage()` might not be precise enough.
Intelephense gives a warning as well:

```
Expected type 'object'. Found 'string|string[]'.
```